src/sql/workbench/contrib/extensions/browser/extensionsActions.ts

```diff
@@ -85,3 +87,58 @@ export class OpenExtensionAuthoringDocsAction extends Action {
 		await this.openerService.open(URI.parse(OpenExtensionAuthoringDocsAction.extensionAuthoringDocsURI));
 	}
 }
+
+export class HidePanel extends Action {
+	static readonly ID = 'workbench.extensions.action.hidePanel';
```

Comment:
This doesn't have anything to do with extensions so I'd prefer we name it something more generic like `workbench.action.hidePanel`

And for now since I don't see us using this anywhere else I'd prefer these also live in the tsgops.contribution.ts file I suggested making above.